math/z3/Portfile

```diff
@@ -295,6 +296,13 @@ subport ${name}-fstar {
     long_description        Private version of the Z3 theorem prover for use by (and qualified to work with) F*.
     cmake.install_prefix    ${prefix}/libexec/${subport}
 
+    set py_ver              3.12
+    set py_ver_nodot        [string map {. {}} ${py_ver}]
+    depends_build-append    port:python${py_ver_nodot} \
```

Comment:
As per lint:
`Warning: Dependency port:python312 specified multiple times in depends_build`
That's because most of this has already been done around line 93.

Reply:
Thank you, I removed duplicates now, hopefully it gonna work.